examples/sample-app/app/wiring/AppLoader.scala

```diff
@@ -20,15 +20,18 @@ class AppComponents(context: Context)
 
   val db = dbApi.database("default")
 
-  val queryRunner = QueryRunner(db.dataSource, implicitly[ExecutionContext])
+  val simpleQueryRunner =
```

Comment:
this could lead to implicit conflicts, if we have two interpreters in the same scope, could we have a way to avoid this ?